src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    check: (state) => {
+      if (state.status === 'Under construction') {
+        state.status = 'Under construction';
+      }
+    },
+  },
+});
+
+export default categorySlice.reducer;
```

Comment:
- [ ] Good job adding `check` 👍, however, let's explore it as an `action`, as it'll be using as an action 👍. We have something like this:

```javascript
   .
   .
   .
export const { check } = categoriesSlice.actions

export default categoriesSlice.reducer
```